tests/test_modeling_flax_roberta.py

```diff
@@ -116,6 +116,8 @@ def prepare_config_and_inputs_for_common(self):
 @require_flax
 class FlaxRobertaModelTest(FlaxModelTesterMixin, unittest.TestCase):
 
+    test_head_masking = True
+
```

Comment:
We should also test BigBird, no ?

Reply:
BigBird is not testing for head masking for `torch` either, as it's required different test script because of various attention mechanisms. Maybe, I can have a look at this in another PR?

Reply:
I see, thanks!